eq-author-api/utils/datastore.js

```diff
@@ -1,302 +1,21 @@
-const jsondiffpatch = require("jsondiffpatch");
-const { omit, first, isNil } = require("lodash");
-const logger = require("pino")();
-const uuid = require("uuid");
-const {
-  QuestionnaireModel,
-  QuestionnaireVersionsModel,
-  dynamoose,
-  justListFields,
-  UserModel,
-  CommentsModel,
-} = require("../db/models/DynamoDB");
+const datastoreDynamoDb = require("./datastore-dynamodb");
```

Comment:
I would move the require to after using the env variable, the reason being is that when you added in firestore then both database modules would be imported.
See https://github.com/ONSdigital/eq-questionnaire-registry/blob/master/database/index.js as an example.
Maybe consider pulling datastore into their own folder too.